xml/security_yast2_security.xml

```diff
@@ -92,12 +92,16 @@
   <title><guimenu>Predefined security configurations</guimenu></title>
 
   <para>
-   &productname; comes with three <guimenu>Predefined Security
+   The includes three <guimenu>Predefined Security
```

Comment:
looks like something got lost here

Reply:
I think you are right. Reverting to the entity, as before, but with the simpler phrasing.